### PR TITLE
Remove now useless proxy install part in setup page

### PR DESCRIPTION
**Pull request by cbosdo**
# Description

Remove deprecated parts for proxy setup page.

# Target branches

Backport targets (edit as needed):

- master 

# Links
- This PR tracks issue #<insert spacewalk issue, if any>
- Related development PR #<insert PR link, if any>

### CODE REVIEW

File: modules/installation-and-upgrade/pages/proxy-container-setup.adoc [hunk 1]
```diff
@@ -70,39 +70,6 @@ This new entry will be shown under previously entered [literal]``Proxy FQDN`` va
 
 Both [literal]``spacecmd``command  and web UI ways generate configuration archive. This archive needs to be made available on container host.
```
Comment (keichwa) [inline modules/installation-and-upgrade/pages/proxy-container-setup.adoc:71]:
```suggestion
Both [command]``spacecmd`` command  and {webui} ways generate a configuration archive.
This archive needs to be made available on container host.
```

File: modules/installation-and-upgrade/pages/proxy-container-setup.adoc [hunk 2]
```diff
@@ -70,39 +70,6 @@ This new entry will be shown under previously entered [literal]``Proxy FQDN`` va
...
-Five {productname} Proxy containers should be present:
-
-- proxy-salt-broker
-- proxy-httpd
-- proxy-tftpd
-- proxy-squid
-- proxy-ssh
-
-And should be part of [literal]``proxy-pod`` container pod.
+Follow the installation instructions to use the archive to get the proxy containers: xref:reference:proxy-container-installation.adoc[].
```
Comment (keichwa) [inline modules/installation-and-upgrade/pages/proxy-container-setup.adoc:75]:
```suggestion
For installation instructions to use the archive to get the proxy containers, see xref:reference:proxy-container-installation.adoc[].
```

**Review by keichwa — CHANGES_REQUESTED** [review]
plus a drive-by edit.

**Review by keichwa — APPROVED** [review]
Thank you!

**Review by jcayouette — APPROVED** [review]

**Pull request merged**